testsuite/model/src/test/java/org/keycloak/testsuite/model/session/SessionTimeoutsTest.java

```diff
@@ -93,9 +93,7 @@ public void cleanEnvironment(KeycloakSession s) {
         InfinispanTestUtil.revertTimeService(s);
         RealmModel realm = s.realms().getRealm(realmId);
         s.getContext().setRealm(realm);
-        UserModel user1 = s.users().getUserByUsername(realm, "user1");
         s.sessions().removeUserSessions(realm);
```

Comment:
Can we remove this line too because of the call to `removeRealm` below?

Reply:
Maybe the whole thing can just rely on the removeRealm() to clean everything up. The providers are fetching the session persister and calling `onRealmRemove` there and the JPA persister removes all sessions already.

Reply:
Yeah, exactly.